setup/insertObjects.js

```diff
@@ -46,8 +111,8 @@ Kompus10_1 = [{
     "Inlopp": "O: 160, typ: PVC rör, VG: 850",
     "Kabelgenomföring": "O: 50, typ: gummitätning, VG: 1800",
     "Antal pumpar": 1,
-    "Pump": "GD-20  Skärande avloppspump, 3-fas",
-    "ArtikelNr": "10-1GD20",
+    "Pump": "GD-20",
+    "Modell": "10-1GD20",
```

Comment:
This should not be named Modell.... A Modell name is already set for Kompus 10_1